src/pipeline/rackhd/docker/Build.groovy

```diff
@@ -0,0 +1,51 @@
+package pipeline.rackhd.docker
+
+def build(String manifest_path, String clone_dir, String workdir){
+    def ret_dict=[:]
+    node(build_docker_node){
+        withEnv([
```

Comment:
sorry... I think we should pass in those things in build() parameter instead of using env var...